metagraph/src/annotation/binary_matrix/multi_brwt/brwt.cpp

```diff
@@ -58,38 +58,38 @@ BRWT::SetBitPositions BRWT::get_row(Row row) const {
     return row_set_bits;
 }
 
-Vector<std::pair<BRWT::Column, uint64_t>> BRWT::get_row_ranks(Row row) const {
-    assert(row < num_rows());
+Vector<std::pair<BRWT::Column, uint64_t>> BRWT::get_column_ranks(Row i) const {
+    assert(i < num_rows());
 
     // check if the row is empty
-    uint64_t rank = nonzero_rows_->conditional_rank1(row);
+    uint64_t rank = nonzero_rows_->conditional_rank1(i);
     if (!rank)
         return {};
 
-    Vector<std::pair<BRWT::Column, uint64_t>> row_ranks;
+    Vector<std::pair<BRWT::Column, uint64_t>> row;
 
     // check whether it is a leaf
     if (!child_nodes_.size()) {
         assert(assignments_.size() == 1);
 
         // the bit is set
         for (size_t j = 0; j < assignments_.size(); ++j) {
-            row_ranks.emplace_back(j, rank);
+            row.emplace_back(j, rank);
         }
-        return row_ranks;
+        return row;
     }
 
     // check all child nodes
     uint64_t index_in_child = rank - 1;
 
-    for (size_t i = 0; i < child_nodes_.size(); ++i) {
-        const auto &child = *child_nodes_[i];
+    for (size_t k = 0; k < child_nodes_.size(); ++k) {
```

Comment:
minor point, maybe call this `j` to avoid confusing it with the k-mer size

Reply:
I use `j` for indexing columns. This is the annotation module, there are no k-mers here.
So `k` is used to index children in a tree (from "k-ary tree")